Controllers/history.js

```diff
@@ -0,0 +1,40 @@
+const { extend } = require("lodash");
+const postHistory = async (req, res) => {
+  let { user } = req;
+  const historyvideo = req.body;
+  const isAllreadyPresent = user.history.filter(
+    (video) => video.id === historyvideo.id
+  );
+
```

Comment:
This doesnt makes sense to me. if the video is already present then it is not getting added but shouldn't it be like time stamp getting updated if it exists? So that order can be maintained>

Reply:
I will resolve this issue, thanks for mentioning